app/views/staff/index.html.erb

```diff
@@ -0,0 +1,81 @@
+
+<section id ="page-contents" >
+
+  <%= form_for 'staff/training_session', :url => { :action => "create", controller: 'staff/training_sessions'} do |f| %>
+
+    <table class="staff-table">
+      <tr>
+        <th>Select</th>
+        <th>Username</th>
+        <th>Last Signed In</th>
+        <th>Location</th>
+        <th>Certification</th>
+      </tr>
+
+    <% @users.each do |user| %>
+      <tr>
+
+        <td class="checkbox-cell">
+          <input type="checkbox" class="user-checkbox" onclick="check_users(this)"></input>
+        </td>
+
+        <td class="username-cell"><%=
+          link_to(content_tag(:username, user.username), user_url(user.username))
+        %></td>
+
+        <td class="signed_in-cell" style="horizontal-alginment:center;"><%=
+          if user.lab_sessions.present?
+            singout_time = user.lab_sessions.last.sign_out_time
+            singout_time.strftime("%B %d, %Y")
+          end
+        %></td>
+
+        <td class="location-cell"><%=
+          user.location
+        %></td>
+
+        <td class="certifications-cell"><%
+          if user.certifications.first.present?
+            user.certifications.each do |certification|
+            %>
+            <div
+             class="cert-name"
+             style="display:inline" >
+             <div class="cert-holder" style="display:inline">
+              <%= certification.training %> |
+              <div id="cert-details" style="display:inline">
+                <%= certification.training %> <br>
+                <%= User.find(certification.trainer_id).name %> <br>
+                <%= certification.updated_at.strftime("%B %d, %Y")  %>
+              </div>
+            </div>
+          </div>
+          <%
+            end
+          end
+          %>
+      </td>
+
+      </tr>
+    <% end %>
+    </table>
+    <br>
+    <input type="hidden", name="training_session[user_id]", value="<%=current_user.id%>"/>
```

Comment:
I wouldn't include this as an input in the form, I would fetch the current user in the controller instead